a = 1
b = '2'

try:
    print(a + b)
except Exception as e:
    print(f"There was an exception: {e}")